app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form526DocumentUploadFailureEmail < Job
+      STATSD_SENT_METRIC_KEY = 'api.form_526.document_upload_failure_notification_sent'
+      STATSD_EXHAUSTED_METRIC_KEY = 'api.form_526.document_upload_failure_email_job_exhausted'
+
+      # retry for one day
+      sidekiq_options retry: 14
+
+      sidekiq_retries_exhausted do |msg, _ex|
+        job_id = msg['jid']
+        error_class = msg['error_class']
+        error_message = msg['error_message']
+        timestamp = Time.now.utc
+        form526_submission_id = msg['args'][0]
+
+        Rails.logger.warn(
+          'EVSS::DisabilityCompensationForm::Form526DocumentUploadFailureEmail retries exhausted',
```

Comment:
I think 'Form 526 Document Upload Failure Email retries exhausted' would suffice.

Reply:
Fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/16359/commits/ecc58aad02d0a63ab765b8da3c7f7b7c0fe1e484